tools/generate_test_status.py

```diff
@@ -130,4 +162,5 @@
     json.dump(report, f, ensure_ascii=False, indent=2)
     f.write("\n")
 
+# Print id to terminal
```

Comment:
We can see the "what" without the comment; it'd be more helpful if the comment told the "why", which isn't necessarily obvious. My guess is that this is for debugging? Or perhaps some other tools is parsing the output of this script?